lib/internal/worker.js

```diff
@@ -415,6 +426,50 @@ function makeResourceLimits(float64arr) {
   };
 }
 
+function eventLoopUtilization(util1, util2) {
+  // TODO(trevnorris): Works to solve the thread-safe read/write issue of
+  // loopTime, but has the drawback that it can't be set until the event loop
+  // has had a chance to turn. So it will be impossible to read the ELU of
+  // a worker thread immediately after it's been created.
+  if (!this[kIsOnline]) {
+    return { idle: 0, active: 0, utilization: 0 };
```

Comment:
If we keep this API, I really prefer it if we didn't return `0`s to indicate "not yet online" here and instead throw an error. I can see users running into footguns here.

Reply:
This currently agrees with the behavior on the main thread. I think throwing an error would be the (way) more unexpected result.